types/mtproto__core/index.d.ts

```diff
@@ -0,0 +1,6307 @@
+// Type definitions for @mtproto/core 6.2
+// Project: https://github.com/alik0211/mtproto-core
+// Definitions by: Viktor Shchelochkov <https://github.com/VityaSchel>, Ali Gasymov <https://github.com/alik0211>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Required for codegen
+// tslint:disable:unified-signatures
+// tslint:disable:max-line-length
+
```

Comment:
Cannot disable this for complete file. Please disable only at the location where there is error so it can ve reviewed to see if its appropriate to disable the lint rule